koku/masu/util/aws/common.py

```diff
@@ -646,7 +646,7 @@ def aws_generate_daily_data(data_frame):
             "lineitem_blendedcost": ["sum"],
             "pricing_publicondemandcost": ["sum"],
             "pricing_publicondemandrate": ["max"],
-            "savingsplan_savingsplaneffectivecost": ["sum"],
+            "calculated_amortized_cost": ["sum"],
```

Comment:
Here we don't want to make a change, as the calculated cost doesn't exist in the raw data. This function just rolls up raw hourly data to daily data.

Reply:
Ok, so where do we need to return `calculated_amortized_cost` then?